### DISCUSSION TITLE
Error on migration

### DISCUSSION THREAD

**Discussion by alexpuertasr** [prisma/prisma#12970] [Q&A] [answered]
Basically I want to merge firstName and lastName into just one column called fullName.
provider = "postgresql"

My schema before:
```prisma
model User {
  id        String     @id @default(cuid())
  firstName String?
  lastName  String?
  createdAt DateTime   @default(now())
  updatedAt DateTime   @updatedAt
}
```
My schema now:
```prisma
model User {
  id        String     @id @default(cuid())
  fullName  String
  createdAt DateTime   @default(now())
  updatedAt DateTime   @updatedAt
}

```
For that I created a custom migration:
(npx prisma migrate dev --name added_full_name --create-only)
```sql
-- AlterTable
ALTER TABLE "User" ADD COLUMN "fullName" TEXT NOT NULL;
UPDATE "User" SET fullName = CONCAT(firstName, " ", lastName);

-- AlterTable
ALTER TABLE "User" DROP COLUMN "firstName";

-- AlterTable
ALTER TABLE "User" DROP COLUMN "lastName";
```
However when I run the migration I get the following error:
(npx prisma migrate dev)
```
Error: P3006

Migration `20220423172725_added_full_name` failed to apply cleanly to the shadow database.
Error:
column "firstname" does not exist
   0: sql_migration_connector::flavour::postgres::sql_schema_from_migration_history
             at migration-engine/connectors/sql-migration-connector/src/flavour/postgres.rs:333
   1: sql_migration_connector::validate_migrations
             at migration-engine/connectors/sql-migration-connector/src/lib.rs:341
   2: migration_core::api::DevDiagnostic
             at migration-engine/core/src/api.rs:111
```

Any idea what I'm doing wrong?

**Comment by austincrim** [comment, marked as answer]
Hey @alexpuertasr 👋 ,

At first glance this does seem like it should work and I reproduced the same error. Would you mind filing a [bug report](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml)? Our engineers will look at it from there.

**Reply by alexpuertasr** [reply]
Ok I see, thank you. I'm going to report this issue

**Comment by Dunsin-cyber** [comment]
Has a fixed been found? I'm getting similar issue here

**Reply by austincrim** [reply]
Hey @alexpuertasr, were you able to get a bug report filed? Do you have the link?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
